### PR TITLE
created ImageCard and started the beginnings of the quick facts secti…

### PR DESCRIPTION
**Pull request by jilly477**
created ImageCard and started the beginnings of the quick facts section on the about change. pushing this to get help with font-size error on description of the imagecard

### CODE REVIEW

**Comment by jilly477** [comment]
I forgot to include in my last commit but I have integrated the imagecard carousel component and finished the quick facts section of the about page

File: components/ImageCardCarousel.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            spaceBetween: 20,
+          },
+          950: {
+            slidesPerView: 2,
+            spaceBetween: 20,
+          },
+        }}
+      >
+        {slides.map((slide) => (
+          <SwiperSlide key={`${slide.title}__swiper__slide`}>
```
Comment (xsharonhe) [inline components/ImageCardCarousel.jsx]: ```title``` does not exist

**Review by xsharonhe — COMMENTED** [review]
Is it possible to centre the card during mobile? I know mine doesn't either but if it's possible...

**Comment by jilly477** [comment]
> Is it possible to centre the card during mobile? I know mine doesn't either but if it's possible...

image
Doesn't your carousel center the slides on mobile here?

**Comment by jilly477** [comment]
I was able to fix the mobile centering issue for all mobile screens except for 'iPhone 5/SE' and 'Galaxy Fold'. This is due to the fact that the background itself is a rounded corner rectangle background and at smaller sizes, the original size of the image does not fit. If I try to make the boxes smaller, it makes the corners sharp because it's the image itself that has the rounded corners. Should I rewrite the code so that the corners are rounded by code ( I was unable to do this before due to some unknown error ) or leave it be for the two mobile sizes?

**Comment by xsharonhe** [comment]
> I was able to fix the mobile centering issue for all mobile screens except for 'iPhone 5/SE' and 'Galaxy Fold'. This is due to the fact that the background itself is a rounded corner rectangle background and at smaller sizes, the original size of the image does not fit. If I try to make the boxes smaller, it makes the corners sharp because it's the image itself that has the rounded corners. Should I rewrite the code so that the corners are rounded by code ( I was unable to do this before due to some unknown error ) or leave it be for the two mobile sizes?

It's probably okay because uh iPhone 5/SE was a while back I think. If it's not entered on those devices it isn't the end of the world. Could you just fix merge conflicts and you can admin privilege them into the  repo.

**Pull request merged**